spanner_orm/field.py

```diff
@@ -50,16 +50,40 @@ class Field(object):
   def __init__(self,
                field_type: Type[FieldType],
                nullable: bool = False,
-               primary_key: bool = False):
+               primary_key: bool = False,
+               length: int = 0,
+               allow_commit_timestamp: bool = False):
     self.name = None
     self._type = field_type
     self._nullable = nullable
     self._primary_key = primary_key
+    self._length = length
+    self._allow_commit_timestamp = allow_commit_timestamp
+
+    if self._length < 0:
+      raise error.ValidationError('length can not be less than zero')
+
+    length_support = ["STRING(MAX)", "ARRAY<STRING(MAX)>", "BYTES(MAX)"]
+
+    if self._type.ddl() not in length_support and self._length:
```

Comment:
Instead, add `supports_length` attribute to the self._type implementation.

Reply:
Done